server/cluster_worker.go

```diff
@@ -65,6 +65,10 @@ func (c *RaftCluster) handleAskSplit(request *pdpb.AskSplitRequest) (*pdpb.AskSp
 		}
 	}
 
+	// Disable merge for the 2 regions in a period of time.
+	c.coordinator.mergeChecker.RecordRegionSplit(reqRegion.GetId())
```

Comment:
is it possible tikv fail to split region after asking split?

Reply:
I think we can accept that false positive case.

Reply:
Why not record that when a region reports split?

Reply:
_AskSplit_ is more reliable, because split must go through this step.